process_pub_database/get_data_cpdl.py

```diff
@@ -8,220 +8,222 @@
 import time
 import unicodedata
 
+from utils import *
```

Comment:
it's not a great idea to import *, because it's difficult to work out where functions come from. Use explicit imports.